modules/node_modules/@ncigdc/components/MutationTable.js

```diff
@@ -118,7 +118,7 @@ const MutationTable: TMutationTable = ({
               </span>
             }
           >
-            # Affected Cases<br /> Across all Projects
+            # Affected Cases<br /> all Projects
```

Comment:
Try not think about it as a sentence (it's table header after all)

It would be best as `# Affected Cases - All Projects` but the line break could also take the place of the hyphen